x-pack/test/functional/apps/endpoint/policy_details.ts

```diff
@@ -13,13 +13,20 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
   // Skipped until we can figure out how to load data for Ingest
   describe.skip('Endpoint Policy Details', function() {
     this.tags(['ciGroup7']);
+    before(async () => {
```

Comment:
this file is confusing me a bit. `before` navigates, then the new test you added at the end navigates in the same way. 
Also, do you happen to know what the existing test is doing?
![image](https://user-images.githubusercontent.com/35559/77553328-0ca6f800-6e8b-11ea-8557-889c0776f703.png)

from the variable names, I don't understand.

Reply:
Just a reminder that these test are currently `skip`'ed and will be completely refactored once we figure out how to load a good set of Ingest data.

Reply:
@oatkiller the variable names are my fault.  I updated this test when I was getting this started.  The names should be changed to `policyDetailsTitle` or something

Reply:
Since these are skipped, maybe we should just remove these instead. If the code is dead, we don't know if it works anyway. Git will have the old code if we ever end up needing it.

Reply:
@oatkiller  Yeah, this is my fault here - you had the same feedback on my PR that `.skip`ed it. I can remove the entire set of files (details + list) and adjust the index file to not load them, but that just seemed like more work than to just leave them in at this time.
Do you have a strong feeling on having it removed?